src/syscall/file_hosted.go

```diff
@@ -0,0 +1,25 @@
+// +build !baremetal,!wasi,!wasm
+
+// This file assumes there is a libc available that runs on a real operating
+// system.
+
+package syscall
+
+const pathMax = 1024
```

Comment:
Note that `PATH_MAX` doesn't really apply nowadays, see: https://insanecoding.blogspot.com/2007/11/pathmax-simply-isnt.html
But I guess it's good enough for now...

Reply:
fwiw, this was copied from the standard library, and different platforms use different values there.